ang/test/civicase/ActivityFeed.spec.js

```diff
@@ -136,6 +136,21 @@
           }
         });
       });
+
+      describe('when the activity details panel is iniliased with window already scrolled', function () {
+        beforeEach(function () {
+          compileDirective(true);
```

Comment:
this `true` value gives little context as to what it means. Can we change the signature of the function so it uses options instead of passing this value directly?

```js
compileDirective({ isAffixedOnInit: true });

function compileDirective (options) {
  // ...
}
```